.github/workflows/main-build.yml

```
@@ -59,7 +59,7 @@ jobs:
 
       - name: Login to GitHub Container Registry
         if: (github.ref == 'refs/heads/main') || (contains(github.event.pull_request.labels.*.name, 'push-image')) || (contains(github.event.pull_request.labels.*.name, 'devnet-test-e2e'))
-        uses: docker/login-action@v3
+        uses: docker/login-action@v4
```

v4 of `docker/login-action` doesn't exist: https://github.com/docker/login-action/releases

```suggestion
        uses: docker/login-action@v3
```